website/cmdlets/index.html

```diff
@@ -93,7 +93,7 @@ <h1 id="google-compute-engine">Google Compute Engine</h1>
 $config = New-GceInstanceConfig "instance-name" -MachineType "n1-standard-4" `
     -DiskImage (Get-GceImage -Family "windows-2012-r2")
 
-$config | Add-GceInstance -Project $myProjectName -Zone "us-central1-b
+$config | Add-GceInstance -Project $myProjectName -Zone "us-central1-b"
```

Comment:
Didn't you or I add this missing quotation mark in another CL? Did we do the merge wrong or something?